lib/openid_connect.ex

```diff
@@ -266,6 +283,43 @@ defmodule OpenIDConnect do
     end
   end
 
+  defp verify_claims(claims, config) do
+    with :ok <- verify_exp_claim(claims, exp_leeway(config)),
+         :ok <- verify_aud_claim(claims, client_id(config)) do
+      {:ok, claims}
+    end
+  end
```

Comment:
Perhaps being able to opt out of these verifications would be useful to some. Certainly on board with secure by default.